docs/proposals/distributed_architecture/README.md

```diff
@@ -0,0 +1,104 @@
+# Liquid Metal Governance Enhancement Proposal
+
+## Enhancement Proposal: Flintlock Distributed Management and Operational Improvements
+
+**Proposal by:** Microscaler
+
+---
+
+## Table of Contents
+
+1. [Executive Summary](#executive-summary)
+2. [Motivation and Background](#motivation-and-background)
+3. [Microscaler's Requirements and Objectives](#microscalers-requirements-and-objectives)
+4. [Detailed Technical Proposals](#detailed-technical-proposals)
+5. [Technical and Community Value](#technical-and-community-value)
+6. [Benefits to the Broader Community](#benefits-to-the-broader-community)
+7. [Implementation Plan and Collaboration](#implementation-plan-and-collaboration)
+8. [Governance and Community Engagement](#governance-and-community-engagement)
+9. [References and Supporting Documentation](#references-and-supporting-documentation)
+
+---
+
+## Executive Summary
+
+Microscaler proposes comprehensive enhancements to Flintlock, significantly advancing distributed management capabilities, operational resilience, and scalability. Key benefits include improved reliability, reduced downtime, enhanced security, and simplified operational practices. Microscaler seeks active collaboration from the Liquid Metal governance board and the broader community to realise these improvements promptly and effectively.
+
+---
+
+## Motivation and Background
+
+Flintlock currently faces scalability, robustness, and operational clarity limitations that hinder effective deployment at scale. Real-world scenarios, such as managing large-scale distributed deployments and recovering from critical host failures, underscore the importance of these enhancements. Addressing these areas ensures Flintlock can effectively support complex, real-world operational needs.
```

Comment:
Flintlock was only ever designed for a single host. It was always envisioned that any distributed scheduling/clustering would be done at a layer above.

I'd be keen to maintain this separation and keep flintlock solely focused on interacting with microvms on a single machine.